libs/sdk-backend-tiger/src/backend/workspace/insights/index.ts

```diff
@@ -39,6 +35,11 @@ import { convertInsight } from "../../../convertors/toBackend/InsightConverter";
 
 import { visualizationClasses as visualizationClassesMocks } from "./mocks/visualizationClasses";
 
+const defaultHeaders = {
```

Comment:
There are the same default headers in the insights section. Should these be shared?

Reply:
yes, I believe they may be shared across all places... do you have any recommandation of the common place where to put them?

Reply:
We can do something like `src/backend/constants.ts` and put it there. We have a similar file in `sdk-backend-bear`.

Reply:
in the end I have moved it in the api-client-tiger (as it's used also in the catalog-export tool)